transformations/dyslexia_words_swap/transformation.py

```diff
@@ -0,0 +1,42 @@
+import os
+import json
+import spacy
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class DyslexiaWordsSwap(SentenceOperation):
```

Comment:
Please add docstrings for class and all the functions.

Reply:
Fixed ! Thanks !